execution-engine/engine-tests/src/test/hdac_system_contracts/pop_install.rs

```diff
@@ -17,11 +17,13 @@ const DEPLOY_HASH_2: [u8; 32] = [2u8; 32];
 const N_VALIDATORS: u8 = 5;
 
 // one named_key for each validator and four for the purses and the total supply amount.
-const EXPECTED_KNOWN_KEYS_LEN: usize = ((N_VALIDATORS * 2) as usize) + 4;
+const EXPECTED_KNOWN_KEYS_LEN: usize = ((N_VALIDATORS * 2) as usize) + 5 + 1;
```

Comment:
Seems to need to correct the comment accodingly

Reply:
https://github.com/hdac-io/CasperLabs/pull/30/commits/653d226b40ca1529d6d067b382f03e42199c967d